From 8d472ea3a37b9aa7ecee59a4a894571cff1d34ae Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 30 Sep 2019 08:45:52 +0200
Subject: [PATCH 46/60] Make default welcome page be blank

Because replacing kChromeUIWelcomeURL with about:blank leads the
browser to load some broken URL instead.
---
 chrome/browser/resources/welcome/welcome.html | 22 -------------------
 1 file changed, 22 deletions(-)

diff --git a/chrome/browser/resources/welcome/welcome.html b/chrome/browser/resources/welcome/welcome.html
index e2a37c5b0a11e..e69de29bb2d1d 100644
--- a/chrome/browser/resources/welcome/welcome.html
+++ b/chrome/browser/resources/welcome/welcome.html
@@ -1,22 +0,0 @@
-<!doctype html>
-<html dir="$i18n{textdirection}" lang="$i18n{language}">
-  <head>
-    <meta charset="utf-8">
-    <title>$i18n{headerText}</title>
-    <link rel="import" href="welcome_app.html">
-    <link rel="stylesheet" href="chrome://resources/css/md_colors.css">
-    <link rel="stylesheet" href="chrome://resources/css/text_defaults_md.css">
-  </head>
-  <body>
-    <style>
-      @media (prefers-color-scheme: dark) {
-        html {
-          background: var(--md-background-color);
-        }
-      }
-    </style>
-    <welcome-app></welcome-app>
-    <script src="/welcome.js"></script>
-    <link rel="stylesheet" href="chrome://welcome/welcome.css">
-  </body>
-</html>
-- 
2.23.0

